### PR TITLE
Branch annotate ug

### PR DESCRIPTION
**Pull request by junjunjieOng**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -478,7 +477,7 @@ The index **must be a positive integer** 1, 2, 3, …​
 
 
 **Examples**:
-* `listt` followed by `deletet 1` lists out all the tasks in NUS Classes, then deletes the task at index 1.
+* `listt all/` followed by `deletet 1` lists out all the tasks in NUS Classes, then deletes the task at index 1.
```
Comment (brian16600) [inline docs/UserGuide.md:480]: Good catch, is there any more incorrect usages of listt throughout?

**Comment by brian16600** [comment]
overall looks good, thanks for the update

File: docs/UserGuide.md (hunk 1 above)
Comment (junjunjieOng) [reply docs/UserGuide.md:480]: Should be no more. But just updated the command summary listt

**Review by snss231 — COMMENTED** [review]
thanks for making the annotations :) lgtm

**Pull request merged**